Project.toml

```diff
@@ -16,13 +16,3 @@ LaTeXStrings = "1.1"
 LightXML = "0.9"
 Luxor = "2"
 julia = "1.4"
-
-[extras]
```

Comment:
Oh we don't need this if we have a `test/Project.toml`? Good to know.

Reply:
Yea! Isn't that cool??? Everything still works fine and it keeps it nicely organized. :)